packages/scheduler/src/forks/SchedulerHostConfig.default.js

```diff
@@ -85,7 +85,7 @@ if (
     // option to rely on it in the future?
     const requestAnimationFrame = window.requestAnimationFrame;
     const cancelAnimationFrame = window.cancelAnimationFrame;
-    // TODO: Remove fb.me link
```

Comment:
This comment looks good to remove. The links it referred to (also added in #14964) were removed in #19598 so it's fine to clean the comment up.